annotations/openshift-annotations/src/main/java/io/ap4k/openshift/confg/OpenshiftConfigCustomAdapter.java

```diff
@@ -17,16 +17,16 @@
 package io.ap4k.openshift.confg;
 
 import io.ap4k.openshift.adapt.OpenshiftConfigAdapter;
-import io.ap4k.openshift.annotation.OpenshiftApplication;
+import io.ap4k.openshift.annotation.GenerateOpenhisftManifest;
```

Comment:
This doesn't seem right :P `GenerateOpenhisftManifest` -> `GenerateOpenshiftManifest`

Reply:
oooooops